package.json

```diff
@@ -11,18 +11,20 @@
     "chimp-test": "chimp chimp.config.js",
     "chimp-test:watch": "yarn run chimp-test --watch",
     "dev": "concurrently \"yarn run dev-client\" \"yarn run dev-server\" \"yarn run dev-parity\"",
-    "dev-parity": "yarn parity",
-    "dev-client": "yarn run clean && yarn run prep && NODE_ENV=test webpack --watch --colors --progress",
-    "dev-server": "NODE_ENV=test nodemon --watch src/server --watch node_modules src/server/index.js localhost:8080 ",
```

Comment:
I'm not sure why this was added in, but I don't think we want `NODE_ENV=test` unless we are actually running tests